src/router/shortUrl.ts

```diff
@@ -13,3 +19,11 @@ router.post("/urls", createUrlValidationRule, validateRequest, postUrl);
 router.get("/urls/:id", getUrlValidationRule, validateRequest, getOneUrl);
 
 router.get("/urls", validateRequest, getAllUrl);
+router.put(
+  "/urls/:id",
+  getUrlValidationRule,
+  createUrlValidationRule,
+  validateRequest,
+  updateOne,
+);
+router.delete("/urls/:id", getUrlValidationRule, validateRequest, deleteOne);
```

Comment:
Give `getUrlValidationRule` a more generic name that would make it work for both get and delete route